photon-core/src/main/java/org/photonvision/vision/calibration/JsonMat.java

```diff
@@ -19,111 +19,57 @@
 
 import com.fasterxml.jackson.annotation.JsonIgnore;
 import com.fasterxml.jackson.annotation.JsonProperty;
-import edu.wpi.first.math.Matrix;
-import edu.wpi.first.math.Num;
-import java.util.Arrays;
-import org.ejml.simple.SimpleMatrix;
-import org.opencv.core.CvType;
+import java.util.Base64;
 import org.opencv.core.Mat;
-import org.opencv.core.MatOfDouble;
-import org.photonvision.common.dataflow.structures.Packet;
+import org.opencv.core.MatOfByte;
+import org.opencv.imgcodecs.Imgcodecs;
 import org.photonvision.vision.opencv.Releasable;
 
 public class JsonMat implements Releasable {
```

Comment:
Is JsonMat still a good name now that the image data is also apart of it?

Reply:
Yep, still fine. JsonMat is consistent with cv::Mat. Images are just matrices